ldaptor/protocols/pureber.py

```diff
@@ -210,10 +209,16 @@ def __init__(self, value=None, tag=None):
         self.value = value
 
     def __str__(self):
-        value = str(self.value)
-        return chr(self.identification()) \
-               + int2berlen(len(value)) \
-               + value
+        if six.PY3 and isinstance(self.value, bytes):
```

Comment:
We should probably default to checking for py2 instead of py3, because eventually we'll have py4...

Reply:
Agreed.